src/prisma/schema.prisma

```diff
@@ -1323,6 +1323,7 @@ model ProposalEvaluation {
   permissions          ProposalEvaluationPermission[]
   proposalNotification ProposalNotification[]
   actionLabels         Json?
+  failReasons          String[]                            @default([])
```

Comment:
lets try doing this w/o adding to evaluation table. i think you do need to save the reason, though?